src/test/java/org/apache/maven/shared/verifier/VerifierTest.java

```diff
@@ -130,4 +139,74 @@ void testDefaultFilterMap() throws VerificationException
         assertTrue( filterMap.containsKey( "@basedir@" ) );
         assertTrue( filterMap.containsKey( "@baseurl@" ) );
     }
+
+    @Test
+    void testDefaultMavenArgument() throws VerificationException
+    {
+        TestVerifier verifier = new TestVerifier( "src/test/resources" );
+
+        verifier.executeGoal( "test" );
+
+        assertThat( verifier.launcher.cliArgs, arrayContaining(
+            "-e", "--batch-mode", "-Dmaven.repo.local=test-local-repo",
+            "org.apache.maven.plugins:maven-clean-plugin:clean", "test" ) );
+    }
+
+    public static Stream<Arguments> argumentsForTest()
+    {
+        return Stream.of(
+            arguments( "test-argument", "test-argument" ),
+            arguments( "test/${basedir}/test", "test/src/test/resources/test" ),
+            arguments( "argument with space", "argument with space" )
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource( "argumentsForTest" )
+    void argumentShouldBePassAsIs(String inputArgument, String expectedArgument) throws VerificationException
```

Comment:
line formatting

Reply:
By the way I'm for checkstyle also for test code ... if someone ask me 😄

Reply:
> By the way I'm for checkstyle also for test code ... if someone ask me 😄

me too